File: 2023/1/part1.py
f = open("input.txt", "r")
input = f.readlines()
f.close()

sum = 0

for line in input:
    nums = []
    for char in line:
        try:
            nums.append(int(char))
        except Exception:
            pass
    sum += 10 * nums[0] + nums[-1]

print(sum)

File: 2015/7/part1.cpp
#include <fstream>
#include <iostream>
#include <string>

int main() {
  std::ifstream file;

  file.open("input.txt");

  if (!file.is_open()) {
    return 1;
  }

  std::string line = "";

  while (std::getline(file, line)) {
    std::cout << line << "\n";
  }

  return 0;
}
